xl2times/config/veda-tags.json

```diff
@@ -1664,6 +1664,18 @@
     "base_tag": "tfm_ins",
     "mod_type": "tsl"
   },
+  {
+    "tag_name": "tfm_ins-txt",
+    "tag_allowed_in": [
+      "BY_Trans",
+      "SR_Trans",
+      "SysSettings",
+      "RegScen",
+      "TradeScen"
```

Comment:
```suggestion
      "SysSettings"
```

Reply:
My understanding is that this is only available in "SysSettings" under Veda. @Antti-L do you have a similar impression?

E.g.: https://forum.kanors-emr.org/showthread.php?tid=1245&highlight=%22tfm_ins-txt%22